content/docs/create-a-new-react-app.md

```diff
@@ -0,0 +1,83 @@
+
+This is also the best way to integrate React into an existing website. You can always add a larger toolchain if you find it helpful!
+
+## Recommended Toolchains
+
+The React team primarily recommends these solutions:
+
+- [Create React App](#create-react-app) (best for [single-page](/docs/glossary.html#single-page-application) apps)
+- [Next.js](#nextjs) (best for Node.js apps)
+- [Gatsby](#gatsby) (best for static websites)
+- [More Flexible Toolchains](#more-flexible-toolchains)
+
+### Create React App
+
+**[Create React App](http://github.com/facebookincubator/create-react-app)** is a comfortable environment for **learning React**, and is the best way to start building **a new [single-page](/docs/glossary.html#single-page-application) application** in React.
+
+It sets up your development environment so that you can use the latest JavaScript features, provides a nice developer experience, and optimizes your app for production. You’ll need to have Node >= 6 and npm >= 5.2 on your machine. To create a project, run:
+
+```bash
+npx create-react-app my-app
+cd my-app
+npm start
+```
+
+>Note
+>
+>`npx` on the first line is not a typo -- it's a [package runner tool that comes with npm 5.2+](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b).
+
+Create React App doesn't handle backend logic or databases; it just creates a frontend build pipeline, so you can use it with any backend you want. Under the hood, it uses [Babel](http://babeljs.io/) and [webpack](https://webpack.js.org/), but you don't need to know anything about them.
+
+When you're ready to deploy to production, running `npm run build` will create an optimized build of your app in the `build` folder. You can learn more about Create React App [from its README](https://github.com/facebookincubator/create-react-app#create-react-app-) and the [User Guide](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#table-of-contents).
+
+### Next.js
+
+**[Next.js](https://nextjs.org/)** is a popular and lightweight framework for **static and server‑rendered applications** built with React. It includes **styling and routing** out of the box, and assumes that you're using [Node.js](https://nodejs.org/) as the server environment.
+
+Learn Next.js from [its official guide](https://nextjs.org/learn/).
+
+### Gatsby
+
+**[Gatsby](https://www.gatsbyjs.org/)** is the best way to create **static websites** with React. It lets you use React components, but outputs pre-rendered HTML and CSS to guarantee the fastest load time.
+
+Learn Gatsby from [its official guide](https://www.gatsbyjs.org/docs/) and a [gallery of starter kits](https://www.gatsbyjs.org/docs/gatsby-starters/).
+
+### More Flexible Toolchains
+
+The following toolchains offer more flexiblity and choice. We recommend them to more experienced users:
```

Comment:
do we really "recommend" all of these? I'm not familiar with all of them but would probably recommend most people turn to webpack+babel first, not sure if I'm skewed

Reply:
Yeah they're all pretty good and do defaults right. Babel/webpack unfortunately don't—people always end up shipping development builds. Webpack 4 might be better on this front but I'd think twice before recommending it directly.